connect/file/src/test/java/org/apache/kafka/connect/file/FileStreamSourceTaskTest.java

```diff
@@ -77,6 +78,17 @@ private void replay() {
 
     @Test
     public void testNormalLifecycle() throws InterruptedException, IOException {
+        normalLifecycle();
+    }
+
+    @Test
+    public void testNormalLifecycleWithResize() throws InterruptedException, IOException {
+        task = new FileStreamSourceTask(2);
```

Comment:
This new test case LGTM. However, it re-create the task. How about setting a small initial size of FileStreamSourceTask to make all test cases meet the resize?

Reply:
Or we make each test create task instead of creating task in ```setup```